tuples.py

```diff
@@ -0,0 +1,47 @@
+# ==========================================
+# Voorbeeld opdracht:
+# Declareer twee tuples met de namen ‘tuple_een’ en ‘tuple_twee’.
+# tuple_een krijgt de waarden 1, 2 en 3. tuple_twee krijgt de waarden 4, 5 en 6.
+# Zorg dat je een tuple krijgt met de waarden 1, 2, 3, 4, 5 en 6. Onthoud dat tuples onveranderlijk zijn.
+# ==========================================
+
+tuple_een = (1, 2, 3)
+tuple_twee = (4, 5, 6)
+
+# je zal een nieuwe tuple variabele moeten declareren. De tuples die al gedeclareerd zijn kunnen niet worden aangepast (zoals een list kan worden aangepast met append, insert en remove).
```

Comment:
Remember dat dit de voorbeeld opdracht is, dus probeer het commentaar iets meer als "verantwoording" te gebruiken. Dus ipv "je zal een nieuwe tuple variabele moeten declareren" meer zoiets als "Omdat tuple_een en tuple_twee niet kunnen worden aangepast, declareren we hier een nieuwe variabele om ze samen te voegen." ofzo